### DISCUSSION TITLE
i will eat leather's nuts

### DISCUSSION THREAD

**Discussion by NotSorena** [Leather128/LeathersFunkinEngine#186] [General]
c

**Comment by Leather128** [comment]
what

**Reply by NotSorena** [reply]
i will eat your nuts

**Reply by Leather128** [reply]
> i will eat your nuts

why do you want to though?

**Reply by NotSorena** [reply]
so you cant have a child

**Reply by Leather128** [reply]
> so you cant have a child

sounds like a good reason

**Reply by JadenMudge** [reply]
uhhhhhhhh ummmmmmm ok

**Reply by MemTheBun** [reply]
TFTWFTFTWFTWTF LEATHER GAY WTFTAFTTFFTA

**Reply by Leather128** [reply]
> TFTWFTFTWFTWTF LEATHER GAY WTFTAFTTFFTA

that's not what i was saying

**Reply by ghost** [reply]
I’m gonna Draw a ship about leather128 and lin0de lol ( ͡° ͜ʖ ͡°)
(is a joke ok)

**Reply by Leather128** [reply]
> I’m gonna Draw a ship about leather128 and lin0de lol ( ͡° ͜ʖ ͡°)
> (is a joke ok)

that is quite the funny joke

**Reply by NotSorena** [reply]
> I’m gonna Draw a ship about leather128 and lin0de lol ( ͡° ͜ʖ ͡°) (is a joke ok)

if you did i would die
because i have a girlfriend (trans 😎)
